pytest/tests/sanity/one_val.py

```diff
@@ -64,6 +67,10 @@
 sent_height = -1
 caught_up_times = 0
 
+# Wait for the killed validator be kicked out, otherwise we will keep missing
+# chunks due to being not validated by that validator.
+utils.wait_for_blocks(nodes[0], target=42)
```

Comment:
We can replace 42 with smth like 3 + epoch_length * 4 to make it more clear. Looks like a matter of taste though.

Reply:
done